XSLT-to-SVG/EmilyKalie/firstSVG.svg

```diff
@@ -26,7 +26,7 @@
         <g> 
             <!-- Eyes -->
             <circle cx="90" cy="90" r="3" fill="black" />
-            <circle cx="110" cy="90" r="3" fill="black" />
+            <circle cx="110" cy="90" r="3" fill="brown" />
```

Comment:
@emikalie I see that you changed the color of your circle for one eye! Cool. [👀](https://emojipedia.org/eyes)